mission_control/navigator_missions/nav_missions/3Ddetect.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
```

Comment:
I think this approach is a better one compared to trying to center the object in the frame since that does not account for rotations or anything like that. The best solution would be to use the lidar to produce a normal vector for the surface and line up with that. But having the 3d pose of the shape is still be preferred
That being said, this file should remain in a feature branch and not here in master